example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -0,0 +1,229 @@
+//
+// Example that demonstrates Offboard velocity control using LOCAL_NED & BODY_NED
```

Comment:
What about?
`Example that demonstrates offboard velocity control in local NED and body coordinates`

Reply:
right..

Reply:
Done.